presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoDriver.java

```diff
@@ -93,7 +91,7 @@ public Connection connect(String url, Properties info)
     public boolean acceptsURL(String url)
             throws SQLException
     {
-        return url.startsWith(DRIVER_URL_START);
+        return PrestoDriverUri.acceptsURL(url);
```

Comment:
from cmt msg:

> Previously we would parse uri with invalid scheme and without jdbc: prefix
that could cause java.lang.StringIndexOutOfBoundsException

this is not quite accurate, since the code was not reachable

also, the cmt title could be improved to suggest the validation is _moved_ not _added:

```
Verify if Presto JDBC url is not malformed within PrestoDriverUri
```